### PR TITLE
Updated traffic management concepts docs

### PR DESCRIPTION
**Pull request by LisaFC**

### CODE REVIEW

File: _docs/concepts/traffic-management/load-balancing.md [hunk 1]
```diff
@@ -41,7 +43,7 @@ the health check API call. In other words, when the number of health
 check failures for a given instance exceeds a pre-specified threshold, it
 will be ejected from the load balancing pool. Similarly, when the number of
 health checks that pass exceed a pre-specified threshold, the instance will
-be added back into the load balancing pool.
+be added back into the load balancing pool. You can find out more about Envoy's failure-handling features in [Handling Failures](./handling-failures.md).
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/load-balancing.md]: failures.md -> failures.html

File: _docs/concepts/traffic-management/manager.md [hunk 2]
```diff
@@ -8,12 +8,8 @@ layout: docs
 type: markdown
 ---
 
-Istio-Manager service is responsible for the lifecycle of
-Envoy instances deployed across the Istio service mesh. It exposes
-APIs for [service discovery](https://lyft.github.io/envoy/docs/configuration/cluster_manager/sds_api.html), dynamic updates to [load balancing pools](https://lyft.github.io/envoy/docs/configuration/cluster_manager/cds.html) and
-[routing tables](https://lyft.github.io/envoy/docs/configuration/http_conn_man/rds.html). These API decouple Envoy from
-platform-specific nuances, simplifying the design and increasing
-portability across platforms.
+The Istio-Manager service is responsible for the lifecycle of
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/manager.md]: We're treating Istio-Manager as a proper name, so "Istio-Manager is responsible..."

File: _docs/concepts/traffic-management/load-balancing.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ layout: docs
 type: markdown
 ---
 
+This page describes how Istio load-balances traffic across instances of a service in a service mesh.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/load-balancing.md]: We generally use "load balancing" with no -

File: _docs/concepts/traffic-management/overview.md [hunk 4]
```diff
@@ -1,44 +1,36 @@
 ---
 title: Overview
-overview: Provides a conceptual overview of traffic management principles in Istio and the kind of features enabled by these principles.
+overview: Provides a conceptual overview of traffic management in Istio and the features enabled by its traffic management principles.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: Doesn't sound great to repeat 'traffic management' twice in one sentence.

**Comment by LisaFC** [comment]
Staged at http://lcarey0-l.dub.corp.google.com:4000/docs/concepts/traffic-management/overview.html

File: _docs/concepts/traffic-management/overview.md [hunk 5]
```diff
@@ -1,44 +1,36 @@
...
 layout: docs
 type: markdown
 ---
 
-Existing container orchestration platforms like Kubernetes, and Mesos, and
-other microservice frameworks allow operators to control when a particular
-set of pods/VMs should receive traffic (e.g., by adding/removing specific
-labels). Unlike existing techniques, Istio decouples traffic flow and infrastructure
-scaling. 
+This page provides an overview of how traffic management works in Istio, including the benefits of its traffic management principles. It assumes that you've already read [What Is Istio?]({{home}}/docs/concepts/what-is-istio/overview.html) and are familiar with Istio's high-level architecture. You can find out more about individual traffic management services and features in the other guides in this section.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: traffic management services and features -> nuke "services and"

File: _docs/concepts/traffic-management/overview.md [hunk 6]
```diff
@@ -1,44 +1,36 @@
...
+
+## Istio-Manager and Envoy
+
+The core component used for traffic management in Istio is the [Istio-Manager]({{home}}/docs/concepts/traffic-management/manager.html) service, which manages and configures all the Envoy proxy instances deployed in a particular Istio service mesh. It lets you specify what rules you want to use to route traffic between Envoy proxies and configure failure recovery features such as timeouts. It also maintains a canonical model of all the services in the mesh and uses this to let Envoys know about the other instances in the mesh via its discovery service.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: is the Istio-Manager -> is Istio-Manager

The link to Istio-Manager can be relative.

File: _docs/concepts/traffic-management/overview.md [hunk 7]
```diff
@@ -1,44 +1,36 @@
...
+
+Each Envoy instance maintains [load balancing information]({{home}}/docs/concepts/traffic-management/load-balancing.html) based on the information it gets from Istio-Manager and periodic health-checks of other instances in its load-balancing pool, allowing it to intelligently distribute traffic between destination instances while following its specified routing rules.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: Relative links

File: _docs/concepts/traffic-management/overview.md [hunk 8]
```diff
@@ -1,44 +1,36 @@
...
+
+## Traffic management benefits
+
+Using this model essentially decouples traffic flow and infrastructure scaling, letting operators specify via Istio-Manager what rules they want service traffic to follow rather than which specific pods/VMs should receive traffic - Istio-Manager and intelligent Envoy proxies look after the rest. So, for example, you can specify via Istio-Manager that you want 5% of traffic for a particular service to go to a canary version irrespective of the size of the canary deployment, or send traffic to a particular version depending on the content of the request.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: Shouldn't use 'this' at the start of a whole new section.

Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: service traffic -> traffic

File: _docs/concepts/traffic-management/request-routing.md [hunk 9]
```diff
@@ -1,28 +1,29 @@
 ---
 title: Request Routing
-overview: Describes how services are modeled within the Istio mesh, the notion of multiple versions of a service, and the communication model between services.
+overview: Describes how requests are routed between services in an Istio service mesh, including the Istio service model and service versions.
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/request-routing.md]: Let's not mention service model here. How about just

Describes how requests are routed between services in an Istio service mesh.

File: _docs/concepts/traffic-management/request-routing.md [hunk 10]
```diff
@@ -1,28 +1,29 @@
...
 type: markdown
 ---
 
-The Istio model of a service is independent of how it is represented
-in the underlying platform (Kubernetes, Mesos, Cloud Foundry,
+This page describes how requests are routed between services in an Istio service mesh.
+
+## Service model and service versions.
+
+As described in [Istio-Manager](./manager.html), the canonical representation of services in a particular mesh is maintained by the Istio-Manager service. The Istio model of a service is independent of how it is represented in the underlying platform (Kubernetes, Mesos, Cloud Foundry,
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/request-routing.md]: by the Istio-Manager service -> by Istio-Manager

File: _docs/concepts/traffic-management/request-routing.md [hunk 11]
```diff
@@ -35,31 +36,33 @@ additional control over traffic between services.
...
+based on the routing rules specified by the operator using Istio-Manager. This model enables
 the application code to decouple itself from the evolution of its dependent
 services, while providing other benefits as well (see
 [Mixer]({{home}}/docs/concepts/policy-and-control/mixer.html)). Routing rules allow Envoy to select a version based
-on criterion such as (headers, url, etc.), tags associated with
-source/destination and/or by weights assigned to each version.
+on criterion such as headers, tags associated with
+source/destination, and/or by weights assigned to each version.
+
+Istio also provides load-balancing for traffic to multiple instances of the same service version. You can find out more about this in [Discovery and Load-Balancing](./load-balancing.html).
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/request-routing.md]: load balancing, no dash

File: _docs/concepts/traffic-management/request-routing.md [hunk 12]
```diff
@@ -35,31 +36,33 @@ additional control over traffic between services.
...
 
-Note that Istio does not provide a DNS. Applications can try to resolve the
+Istio does not provide a DNS. Applications can try to resolve the
 FQDN using the DNS service present in the underlying platform (kube-dns,
 mesos-dns, etc.).
 
 ## Ingress and Egress Envoys
 
-Istio assumes that all traffic entering and leaving the istio service mesh
+Istio assumes that all traffic entering and leaving the Istio service mesh
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/request-routing.md]: the Istio service mesh -> the service mesh

File: _docs/concepts/traffic-management/request-routing.md [hunk 13]
```diff
@@ -35,31 +36,33 @@ additional control over traffic between services.
...
 transits through Envoy proxies. By deploying the Envoy proxy in front of
 services, operators can conduct A/B testing, deploy canary services,
 etc. for user-facing services. Similarly, by routing traffic to external
-web services (e.g., accessing Maps API, or a video service API) via Envoy,
+web services (for instance, accessing the Maps API, or a video service API) via Envoy,
 operators can add failure recovery features such as circuit breakers,
-impose rate limits via Mixer, and provide authentication using
+impose rate limits via Mixer, and provide authentication using the
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/request-routing.md]: the Istio-Auth service -> Istio-Auth

File: _docs/concepts/traffic-management/overview.md (hunk 6 above)
Comment (rshriram) [inline _docs/concepts/traffic-management/overview.md]: such as timeouts, retries and circuit breakers

File: _docs/concepts/traffic-management/overview.md [hunk 14]
```diff
@@ -1,44 +1,36 @@
...
+
+## Istio-Manager and Envoy
+
+The core component used for traffic management in Istio is the [Istio-Manager]({{home}}/docs/concepts/traffic-management/manager.html) service, which manages and configures all the Envoy proxy instances deployed in a particular Istio service mesh. It lets you specify what rules you want to use to route traffic between Envoy proxies and configure failure recovery features such as timeouts. It also maintains a canonical model of all the services in the mesh and uses this to let Envoys know about the other instances in the mesh via its discovery service.
+
+Each Envoy instance maintains [load balancing information]({{home}}/docs/concepts/traffic-management/load-balancing.html) based on the information it gets from Istio-Manager and periodic health-checks of other instances in its load-balancing pool, allowing it to intelligently distribute traffic between destination instances while following its specified routing rules.
+
+## Traffic management benefits
```
Comment (rshriram) [inline _docs/concepts/traffic-management/overview.md]: I would spruce this up. It's not a benefit. This is our fundamental philosophy, different from the traffic management philosophies of other platforms

File: _docs/concepts/traffic-management/overview.md (hunk 6 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done

File: _docs/concepts/traffic-management/overview.md (hunk 7 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done.

File: _docs/concepts/traffic-management/overview.md (hunk 8 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Well-spotted, I think I wrote that before I put the section heading in. Done.

Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md (hunk 9 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/request-routing.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md (hunk 10 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/request-routing.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md (hunk 11 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/request-routing.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md (hunk 12 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/request-routing.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md (hunk 13 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/request-routing.md]: Done.

File: _docs/concepts/traffic-management/load-balancing.md (hunk 3 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/load-balancing.md]: Done.

File: _docs/concepts/traffic-management/load-balancing.md (hunk 1 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/load-balancing.md]: Done.

File: _docs/concepts/traffic-management/manager.md (hunk 2 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/manager.md]: Done.

File: _docs/concepts/traffic-management/overview.md (hunk 4 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md [hunk 15]
```diff
@@ -12,11 +12,15 @@ This page describes how requests are routed between services in an Istio service
 
 ## Service model and service versions.
 
-As described in [Istio-Manager](./manager.html), the canonical representation of services in a particular mesh is maintained by the Istio-Manager service. The Istio model of a service is independent of how it is represented in the underlying platform (Kubernetes, Mesos, Cloud Foundry,
+As described in [Istio-Manager](./manager.html), the canonical representation
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/request-routing.md]: How about:

[Istio-Manager] describes how the manager maintains the canonical representation of services within the mesh.

File: _docs/concepts/traffic-management/overview.md (hunk 5 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done.

File: _docs/concepts/traffic-management/request-routing.md (hunk 15 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/request-routing.md]: "the manager" sounds odd there, as we've never used that term to describe Istio-Manager - I know the duplication of Istio-Manager sounds a bit clunky but I think the alternatives are worse.

File: _docs/concepts/traffic-management/overview.md (hunk 14 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: How about changing it to "Traffic management principles"?

(Also obviously it is our philosophy, but it *provides* benefits, and I think that's what users are going to care about. IME, users are generally not particularly interested, even in a conceptual overview, about why development teams made particular design decisions or their abstract design goals, they just want to know how the thing works and why they should use it - it's what distinguishes a user-facing conceptual doc from a design doc or RFC. I know we're also trying to recruit contributors but even still...)

File: _docs/concepts/traffic-management/load-balancing.md [hunk 16]
```diff
@@ -27,19 +30,21 @@ dynamically update their load balancing pools accordingly.
...
 In addition to load balancing, Envoy periodically checks the health of each
 instance in the pool. Envoy follows a circuit breaker style pattern to
 classify instances as unhealthy or healthy based on their failure rates for
 the health check API call. In other words, when the number of health
 check failures for a given instance exceeds a pre-specified threshold, it
 will be ejected from the load balancing pool. Similarly, when the number of
 health checks that pass exceed a pre-specified threshold, the instance will
-be added back into the load balancing pool.
+be added back into the load balancing pool. You can find out more about Envoy's
+failure-handling features in [Handling Failures](handling-failures.html).
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/load-balancing.md]: handling-failures.html -> ./handling-failures.html.

**Comment by ldemailly** [comment]
build error is:
```
- ./_site/docs/concepts/what-is-istio/overview.html
  *  internally linking to /docs/concepts/goals.html, which does not exist (line 1)
     <a href="/docs/concepts/goals.html">Design Goals</a>
```

File: _docs/concepts/traffic-management/load-balancing.md (hunk 16 above)
Comment (geeknoid) [inline _docs/concepts/traffic-management/load-balancing.md]: Do we want a - in "failure-handling"?

File: _docs/concepts/traffic-management/overview.md [hunk 17]
```diff
@@ -1,41 +1,59 @@
...
 
-Existing container orchestration platforms like Kubernetes, and Mesos, and
-other microservice frameworks allow operators to control when a particular
-set of pods/VMs should receive traffic (e.g., by adding/removing specific
-labels). Unlike existing techniques, Istio decouples traffic flow and infrastructure
-scaling. 
+This page provides an overview of how traffic management works
+in Istio, including the benefits of its traffic management
+principles. It assumes that you've already read [What Is
+Istio?]({{home}}/docs/concepts/what-is-istio/overview.html) and are familiar
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: I think the link needs to all be on one line for markdown to be processed correctly.

File: _docs/concepts/traffic-management/overview.md [hunk 18]
```diff
@@ -1,41 +1,59 @@
...
+with Istio's high-level architecture. You can find out more about individual
+traffic management features in the other guides in this section.
+
+## Istio-Manager and Envoy
+
+The core component used for traffic management in Istio is
+[Istio-Manager](manager.html), which manages and configures all the Envoy
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: ./manager.html

File: _docs/concepts/traffic-management/overview.md [hunk 19]
```diff
@@ -1,41 +1,59 @@
...
+
+Each Envoy instance maintains [load balancing information](load-balancing.html)
+based on the information it gets from Istio-Manager and periodic health-checks
+of other instances in its load-balancing pool, allowing it to intelligently
+distribute traffic between destination instances while following its specified
+routing rules.
+
+## Traffic management benefits
+
+Using Istio's traffice management model essentially decouples traffic flow
```
Comment (geeknoid) [inline _docs/concepts/traffic-management/overview.md]: traffice-> traffic

File: _docs/concepts/traffic-management/overview.md (hunk 17 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Gah, this is yet another reason why I hate Markdown - HTML still works if you line-wrap it! Done.

File: _docs/concepts/traffic-management/overview.md (hunk 19 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done.

File: _docs/concepts/traffic-management/overview.md (hunk 18 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/overview.md]: Done.

File: _docs/concepts/traffic-management/load-balancing.md (hunk 16 above)
Comment (LisaFC) [inline _docs/concepts/traffic-management/load-balancing.md]: Yes, it's a compound adjective. Also, done.

**Comment by ldemailly** [comment]
looks like you missed my comment showing the bad link introduced by this PR

**Pull request merged**